express-notes-api/index.js

```diff
@@ -0,0 +1,121 @@
+const express = require('express');
+const fs = require('fs');
+const notes = require('./data')
+
+const app = express();
+
+function write(input){
+  fs.writeFile('data.json', input, (err) => {
+    if (err) throw err;
+    console.log('File write successful');
+  });
+}
+
+
+app.get('/api/notes', (req, res) =>{
+  let notesSend = []
+
+  for (let note in notes){
+    notesSend.push(notes[note])
+  }
+  res.status(200).json(notesSend);
+})
+
+
+app.get('/api/notes/:id', (req, res, next) => {
+
+  if (Number.isInteger(Number(req.params.id))  && Number(req.params.id) > 0){
+
+
+    if (String(req.params.id) in notes.notes){
```

Comment:
all `req.params` properties are strings, so you can leave out this `String()` coercion. Check the rest of the code for it too.